src/index.js

```diff
@@ -12,24 +11,18 @@ const DEFAULT_ACCUMULATOR = [];
  * @param  {Any}      [options.accumulator] Reducer accumulator
  * @return {Any}
  */
-export const pageTiming = ({metrics = performanceMetrics, reducer = DEFAULT_REDUCER, accumulator = DEFAULT_ACCUMULATOR} = {}) =>
+export const pageTiming = ({metrics, reducer = DEFAULT_REDUCER, accumulator = []} = {}) =>
```

Comment:
Completely unrelated to this specific change, but while we're at it.
I believe it would be safer if we forced the consumer to pass both `reducer` and `accumulator` because they are interdependent.

Currently, there may be a situation in which the consumer passes a reducer that accumulates into an object, but relies on the default accumulator array.

I would consider changing it to something like
```js
({metrics, reduce = { reducer = DEFAULT_REDUCER, accumulator = []}} = {}) => {...}
```
wdyt?

Reply:
That's not 100% correct.
I think it will be pretty common to use an array as an iterator, accumulator is not needed (defaults to empty list)

I agree it is unlikely, but you could reduce onto an existing list:
```js
const event = {
  type: 'my_type',
  metrics: [
    ['custom_event_1', 300], ['custom_event_2', 330]
  ]
};
pageTiming({accumulator: event.metrics})
```

You could also ignore the accumulator (and the functions' result) altogether
```js
const event = {
  type: 'my_type',
  metrics: {}
};
pageTiming({
  reducer: (_, [key, value]) => {...event.metrics, {[key]: value}}
})
```

But really, I don't appreciate for nested named arguments (`options.options`). The main reason I chose named arguments is to make the function signature self explanatory. This way I feel there's another interface to understand